localization/english/event_localization/religion_events/wc_religious_interaction_events_l_english.yml

```
@@ -0,0 +1,29 @@
+﻿l_english:
+ # Religious Head of Faith Interactions
+ wc_religious_interaction.1010:0 "After much deliberation I have decided to repent for my poor deeds. For the Inquisition to mark me as a heretic was entirely pertinent.\n\nI know that my shortcomings caused me to become marked, but those qualities will not outshine me as a follower of [actor.GetFaith.HighGodName]. If the Inquisition, by the grace of [actor.GetFaith.HighGodName], decided to have some mercy on me, I would be most grateful."
+ wc_religious_interaction.1010.accept:0 "Very well, [actor.GetName], we accept your surrender."
+ wc_religious_interaction.1010.refuse:0 "You are not yet worthy of redemption, [actor.GetName]..."
+ wc_religious_interaction.1011:0 "We have received your letter, [actor.GetTitledFirstName]. The Inquisition is grateful for your willing surrender. Agents are already on their way to take you into custody, and you are expected to cooperate with them to the greatest of your ability.\n\nKnow that this will not spare you from the rigors of true penance, but because you chose not to flee or fight the Inquisition may win you the trust of some."
+ wc_religious_interaction.1011.cynical:0 "Let's get this over with."
+ wc_religious_interaction.1011.zealous:0 "Do your worst, inquisitors, faith is my strength."
+ wc_religious_interaction.1011.deceitful:0 "I knew it would come to this, I am prepared."
+ wc_religious_interaction.1011.arrogant:0 "Your inquisitors cannot break me."
+ wc_religious_interaction.1011.normal:0 "I know that I am innocent, as will they."
+ wc_religious_interaction.1012:0 "We have received your letter, [actor.GetTitledFirstName]. The Inquisition has chosen not to accept your surrender. Spoken plainly, we don't intend to allow the guilty to choose their own terms. Regardless, one who cares for his soul would cooperate when the time comes."
+ wc_religious_interaction.1012.deceitful:0 "They must suspect a trap."
```

```suggestion
 wc_religious_interaction.1012.deceitful:0 "They must suspect a trap."
```

this does not make sense

---

Why ?